### DISCUSSION TITLE
How to change the sphinx_research's color in dark mode

### DISCUSSION THREAD

**Discussion by mendax1234** [pradyunsg/furo#469] [Potential Issue] [answered]
![image](https://user-images.githubusercontent.com/62934294/176132852-0254e514-e4ca-4b8b-90dc-8df699a6662e.png)
In dark mode ,the sphinx_research's is still white. Is there any method to change it? @pradyunsg

**Comment by pradyunsg** [comment, marked as answer]
This one slipped through the cracks. :)

> Is there any method to change it? @pradyunsg

Other that writing [custom CSS](https://pradyunsg.me/furo/customisation/injecting/) for it, not that I know of sadly.
